.circleci/config.yml

```diff
@@ -163,7 +163,7 @@ jobs:
             command: |
               bash -c "$PACKCHECK cabal"
         - *save
-  cabal-ghc-9_8_1-docspec:
+  cabal-ghc-9_4_8-docspec:
```

Comment:
Was it working earlier with 9.8? When did it start failing?

Reply:
No, I was just playing around with the versions. The docspec is still failing.